tests/test_smoke.py

```diff
@@ -182,16 +183,37 @@ def run_one_test(test: Test) -> Tuple[int, str, str]:
 
 
 def get_aws_region_for_quota_failover() -> Optional[str]:
-
     candidate_regions = AWS.regions_with_offering(instance_type='p3.16xlarge',
                                                   accelerators=None,
                                                   use_spot=True,
                                                   region=None,
                                                   zone=None)
 
     for region in candidate_regions:
-        if not AWS.check_quota_available(
-                region=region.name, instance_type='p3.16xlarge', use_spot=True):
+        resources = sky.Resources(cloud=sky.AWS(),
+                                  instance_type='p3.16xlarge',
+                                  region=region.name,
+                                  use_spot=True)
+        if not AWS.check_quota_available(resources):
+            return region.name
+
+    return None
+
+
+def get_gcp_region_for_quota_failover() -> Optional[str]:
+
+    candidate_regions = GCP.regions_with_offering(instance_type=None,
+                                                  accelerators={'A100': 1},
```

Comment:
Can we change this to `A100-80GB` here, at L215, L2335 and L2342? They are much harder to find quotas for :)